spring-support-root/spring-mvc-endpoint-support/src/main/java/org/divy/common/bo/endpoint/WebEndpointConfig.java

```diff
@@ -7,11 +7,13 @@
 import org.divy.common.bo.rest.EndPointRegistry;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.divy.common.bo.rest.EndPointRegistry'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)